### ISSUE TITLE
UI/UX website design revamp request 

### ISSUE THREAD

**Issue by techietonic** [KwickerHub/frontend#292]
I am reaching out to express our interest in revamping the UI/UX design of your website. I'd like to work on navigation, accessibility, visual appeal. But if you would want a total design revamp, I can work on that too.
I also believe that an updated design will enhance the user experience and overall usability.

Thanks in advance for considering my request

**Comment by NtemKenyor** [comment]
Yes we need a UI/UX designer and we can start from the dashboard because it needs a lot of corrections and improvements. Can we jump on a call? +2349097441316

**Comment by A1ya1** [comment]
> Yes we need a UI/UX designer and we can start from the dashboard because it needs a lot of corrections and improvements. Can we jump on a call? +2349097441316

I'd prefer a call in discord. I sent you a dm
